vue-app/src/components/Cart.vue

```diff
@@ -49,14 +49,13 @@
           <div>Choose some projects that you want to contribute to...</div>
           <router-link to="/projects" class="btn-secondary mobile mt1">See projects</router-link>
         </div>
-        <div v-else-if="$store.getters.canUserReallocate && !isCartEmpty">
+        <div v-else-if="$store.getters.canUserReallocate && !isCartEmpty && !$store.getters.hasReallocationPhaseEnded">
           <div class="flex-row-reallocation">
             <div class="semi-bold">{{editModeSelection ? "Edit contributions" : "Your contributions"}}</div>
-            <div class="semi-bold" @click="handleEditState" v-if="$store.getters.canUserReallocate">
-              <span v-if="editModeSelection">Cancel</span>
-              <span v-else>Edit</span>
+            <div class="semi-bold" v-if="$store.getters.canUserReallocate">
+              <button @click="handleEditState">{{ editModeSelection ? "Cancel" : "Edit" }}</button>
             </div>
-            <!-- <div @click="removeAll" v-if="$store.getters.canUserReallocate">Remove all</div> -->
+            <!-- <div v-if="editModeSelection" @click="removeAll">Remove all</div> -->
```

Comment:
@samajammin the Remove All (not the one that lives in the dropdown) was commented by Ryan, so I leave it like that. As we discussed before, I think we can leave it in that state and tackle it down in a separate PR.

Reply:
Yep, let's remove that.